src/tasks/installers/operator.ts

```diff
@@ -60,6 +61,66 @@ export class OperatorTasks {
           }
         }
       },
+      {
+        title: 'Checking for pre-created TLS secret',
+        // In case of Openshift infrastructure the certificate from cluster router will be used, so no need in the `che-tls` secret.
+        skip: () => !isKubernetesPlatformFamily(flags.platform),
+        task: async (_: any, task: any) => {
+          // Che is being deployed on Kubernetes infrastructure
+
+          if (!(flags.tls || await this.checkCrForTls(flags))) {
+            // No TLS mode, skip this check
```

Comment:
`flag.tls` is ignored if CR file is used.
so basically `flag.tls` make sense when `flags['che-operator-cr-yaml']` is not defined.

Reply:
I see what you mean. Will change the priority.